spec/controllers/repositories_controller_spec.rb

```diff
@@ -22,7 +25,10 @@
   describe "GET #show" do
 
     it "assigns the requested repository as @repository" do
-      repository = create(:repository)
+      team = create(:team)
+      TeamUser.create(team: team, user: user, role: TeamUser.roles["owner"])
+      namespace = create(:namespace, team: team)
+      repository = create(:repository, namespace: namespace)
```

Comment:
I think this part can be refactored because it's common to the two tests.

I would define a `let(:visible_repository) { create(:repository, namespace: namespace) }` and all the other objects involved at the top of the spec.